deploy/iso/minikube-iso/package/containerd-bin/containerd-bin.mk

```diff
@@ -3,9 +3,9 @@
 # containerd
 #
 ################################################################################
-CONTAINERD_BIN_VERSION = v1.4.9
-CONTAINERD_BIN_COMMIT = e25210fe30a0a703442421b0f60afac609f950a3
-CONTAINERD_BIN_SITE = https://github.com/containerd/containerd/archive
+CONTAINERD_BIN_VERSION = v1.5.8
+CONTAINERD_BIN_COMMIT = 1e5ef943eb76627a6d3b6de8cd1ef6537f393a71
+CONTAINERD_BIN_SITE = https://github.com/containerd/containerd/releases
```

Comment:
https://github.com/containerd/containerd/releases/v1.4.9.tar.gz results in `Not Found`

Reply:
Thanks, I see that "https://github.com/containerd/containerd/archive" is working with a version parameter.